src/components/textarea/textarea.scss

```diff
@@ -75,8 +76,10 @@ label {
     font-weight: var(--font-weight-500);
     letter-spacing: 0;
     line-height: 20px;
+    font-family: inherit;
     cursor: text;
     display: inline-block;
+    color: $app-primary-black;
```

Comment:
Can we not use Sass variables wherever possible. You can use the freshworks colors. We plan to slowly remove this.